src/constants/CardPropTypes.js

```diff
@@ -71,6 +71,9 @@ export const ValueCardPropTypes = {
     extraTooltipText: PropTypes.string,
     learnMoreElement: PropTypes.element,
   }),
+  cardVariables: PropTypes.objectOf(
+    PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.number])
```

Comment:
need to document the function args of the callback function somewhere (maybe in the story) so users of the cardVariables function option know what the callback args are